handout/a11y/key-concerns/visual-assistance.md

```diff
@@ -0,0 +1,9 @@
+# Visual Assistance
+
+## Color Contrast
+
+One category of disability are people åwho are color blind or deficient. This means that when choosing colors to represent different elements on the website, the contrast between them needs to be considered. For WCAG 2.0 AA, this means that the contrast ratio for text or visual representations of text needs to be at least [4.5:1](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast). The contrast ratio of two different colors is not really obvious so there are tools online to measure the contrast ratio such as this [color contrast checker](http://webaim.org/resources/contrastchecker/) from WebAIM or be checked with using automation tests.
```

Comment:
Colour vs color below.

> One category of disability are people who are color blind or _colour?_ deficient. 

Paragraph break.

> This means that when ~~choosing~~ _chosing_ colors to represent different elements on the website, the contrast between them needs to be considered. 

> For WCAG 2.0 AA, this means that the contrast ratio for text or visual representations of text needs to be at least [4.5:1](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast). 

Paragraph break.

> ~~The contrast ratio of two different colors is not really obvious so there~~ _There_ are _online_ tools ~~online~~ to measure the contrast ratio such as this [color contrast checker](http://webaim.org/resources/contrastchecker/) from WebAIM.

Paragraph break.

> ~~ or be checked with using automation tests.~~

Above, maybe provide a link to a library that can be integrated into a testing suite instead, with an example usage. Maybe not super specific, but demonstrate the general idea. How a threshold could be set that would fail if the ratio isn't met.

Also, consider adding an image of a typical problematic image for people with colour blindness. 

Finally, demonstrate how any data expressed with colours should also be expressed with patterns (or some other means to distinguish one data point from another), ie, a bar chart, pie chart, map with states/provinces in different colours.

https://www.smashingmagazine.com/2016/06/improving-ux-for-color-blind-users/#11-graphs

That's actually a really good article you might want to link to, but specifically there's an image of what I mean.